### PR TITLE
Addition to the #579

### PR DESCRIPTION
**Pull request by TikhomirovSergey**
## Change list

The chaining of the activity parameter setting.
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
@email2vimalraj You could review it too.

File: src/test/java/io/appium/java_client/android/IntentTest.java [hunk 1]
```diff
@@ -47,26 +49,25 @@
...
+        assertTrue(((Predicate<AndroidDriver>) driver -> {
+            final Activity activity = new Activity("com.android.mms", ".ui.ComposeMessageActivity");
+            activity.setIntentAction("android.intent.action.SEND");
+            activity.setIntentCategory("android.intent.category.DEFAULT");
+            activity.setIntentFlags("0x4000000");
+            activity.setOptionalIntentArguments("-d \"TestIntent\" -t \"text/plain\"");
+            driver.startActivity(activity);
+            return true;
+        }).test(driver));
+
```
Comment (mykola-mokhnach) [inline src/test/java/io/appium/java_client/android/IntentTest.java]: extra line here

**Comment by SrinivasanTarget** [comment]
Nice Idea @TikhomirovSergey

File: src/test/java/io/appium/java_client/android/IntentTest.java [hunk 2]
```diff
@@ -47,26 +49,25 @@
...
-        }
+    @Test public void startActivityWithIntent() throws Exception {
+        assertTrue(((Predicate<AndroidDriver>) driver -> {
+            final Activity activity = new Activity("com.android.mms", ".ui.ComposeMessageActivity");
+            activity.setIntentAction("android.intent.action.SEND");
+            activity.setIntentCategory("android.intent.category.DEFAULT");
+            activity.setIntentFlags("0x4000000");
+            activity.setOptionalIntentArguments("-d \"TestIntent\" -t \"text/plain\"");
+            driver.startActivity(activity);
+            return true;
```
Comment (mykola-mokhnach) [inline src/test/java/io/appium/java_client/android/IntentTest.java]: is the whole construction with _assertTrue_ and _return true_ really necessary?

Comment (TikhomirovSergey) [inline src/test/java/io/appium/java_client/android/IntentTest.java]: @mykola-mokhnach 
Yes. It is for codecy checkings. It requires assertions. Also I agree that assertions inside tests methods is the good style.

**Comment by TikhomirovSergey** [comment]
@SrinivasanTarget @mykola-mokhnach I will fix the issue later.

**Comment by email2vimalraj** [comment]
@TikhomirovSergey : Good idea.

Kindly update the `The-starting-of-an-Android-app.md` file and the javadoc on the `StartsActivity.java` interface.

**Pull request merged**